index.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" type="text/css" href="./style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>Portifolio | Stanley</title>
+</head>
+<body>
+  <!--Navigation bar--> 
+  <header>
+    <a href="#home" class="logo">Ezeoke Stanley</a>
+    <nav>
+         <img src="./icons/menu.png" alt="envelope icon" class="envelope">
+     </nav>
+</header>
+         <!--Header-->
+         <section class="container">
+           <div class="about">
+             <h1>Hey There.<br>
+              I'm Stan</h1>
+             <h3 class="H-title">I'am a Software Developer</h3>
+             <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+              don’t hestiate to contact me.</p>
+           </div>
+              <!--Social media icon-->
+              <div class="social-media">
+                  <ul>
+                    <li><a href="#yourgithublink"><img
+                        src="icons/github.png"
+                        class="icon"
+                        alt="github icon"
+                        id="github"
+                      /></a>
+                    </li>
+                    <li><a href="#yourlinkedln"><img src="icons/linkedln.png" 
+                      class="icon" alt="linkedin icon" /></a>
+                    </li>
+                    <li><a href="#yourhandlink"><img
+                        src="icons/hand.png"
+                        class="icon right"
+                        alt="hand icon"
+                        id="hand"/></a>
+                    </li>
+                    <li><a href="#yourtwitter">
+                      <img src="icons/twitter.png"
+                        class="icon right"
+                        alt="twitter icon"
+                        id="twitter"
+                      /></a>
+                    </li>
+                    <li><a href="#yourmedium">
+                      <img src="icons/m.png" 
+                      class="icon" 
+                      alt="medium icon" 
+                      id="medium" /></a>
+                    </li>
+                  </ul>
+                </div>
+              </section>
+            </body>
+          </html>
```

Comment:
- [ OPTIONAL] your design looks good, but please make this change for it to look better, this review is optional since the previous reviewer did not mention it, but I strongly recommend you make this change as you move onto the next project, just some few adjustments, you see? 😄  alright, let's go and fix that
<img width="392" alt="optional" src="https://user-images.githubusercontent.com/79058364/159928938-591dec70-89a2-46eb-a3b2-3dffd116be91.png">